.travis.yml

```diff
@@ -28,3 +39,9 @@ script:
 
 after_success:
   - coveralls
+  - docker login -e $DOCKER_EMAIL -u $DOCKER_USER -p $DOCKER_PASS
```

Comment:
Got it. Thus maybe it should be on `before_script` section, so when you got to the `script` (the tests) section, the Docker image is ready…

If these commands take a while to exit, take a look on [`travis_wait`](https://docs.travis-ci.com/user/common-build-problems/#Build-times-out-because-no-output-was-received).